massa-models/src/address.rs

```diff
@@ -681,16 +681,16 @@ mod test {
         // println!("sc_addr_1: {}", sc_addr_1);
 
         // let v1 = "AU12M3AQqs7JH7mSe1UZyEA5NQ7nGQHXaqqxe1TGEpkimcRhsQ4eF";
-        let v2 = "AU4cJWyjpBetGwaRqFDXyrHiQuGB3QKrwjzGiGSzQPGeAARB9AY4";
-        let addr = Address::from_str(v2).unwrap();
+        // let v2 = "AU4cJWyjpBetGwaRqFDXyrHiQuGB3QKrwjzGiGSzQPGeAARB9AY4";
```

Comment:
This kind of comments are only causing confusion to new readers and should always be removed. Git history is there if they need to see the modifications

Reply:
Done